### PR TITLE
Update to latest Bootstrap CSS, JS, and docs HTML

### LINKED ISSUE #543: Navbar collapse menu not working on icons.getbootstrap.com
Navigation bar Menu collapse on chrome for Android doesn't work on new website of bootstrap V.5.0.0

### PR DESCRIPTION
**Pull request by mdo**
Replaces #547.

- Updates to the beta CSS
- Includes the beta JS to fix broken docs (fixes #543)
- Modifies the navbar HTML to match main site

Stop-gap measure until we figure out how best to pull in parts of our docs layout and assets without copy-pasta-ing.

### CODE REVIEW

**Comment by XhmikosR** [comment]
This should also drop purge css from the npm deps and npm scripts.

File: docs/layouts/partials/scripts.html [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {{- if .IsHome }}
 <script src="/assets/js/list.min.js"></script>
+<script src="/assets/js/bootstrap.min.js"></script>
```
Comment (XhmikosR) [inline docs/layouts/partials/scripts.html:3]: Might be worth checking if we can use async here and if there are any console errors and everything works correctly.

**Comment by mdo** [comment]
I'm still running PurgeCSS, so it'll stay until we replace this with an entirely new solution.

**Comment by XhmikosR** [comment]
@mdo the navbar transition is broken probably due to purgecss removing the needed classes. We need to exclude those.

**Pull request merged**